canvas_modules/common-canvas/src/common-properties/components/title-editor/title-editor.jsx

```diff
@@ -65,8 +82,33 @@ class TitleEditor extends Component {
 			</Button>)
 			: null;
 
+		let subtitle = null;
+		if (this.props.subtitle && this.props.uiHints && (this.props.uiHints.label || this.props.uiHints.icon)) {
```

Comment:
How would someone do translations?  I think this needs to get converted into form.json then passed to common-properties.

Reply:
Fixed to use translations in Form.js, thanks!